test/test-extensions.js

```diff
@@ -331,11 +331,16 @@
     }
   }
 
+  /**
+   * @typedef {Object} xmleqlOptions
```

Comment:
Minor, but in #11977 I suggested we change this to `XMLEqlOptions` for consistency with other custom types.